drivers/hmis/app/models/hmis/hud/enrollment.rb

```diff
@@ -281,6 +281,68 @@ def self.generate_enrollment_id
     generate_uuid
   end
 
+  # Data Collection Features that are enabled for this enrollment, thru its project (e.g. Current Living Situation)
+  #
+  # Is it enabled?
+  #   If ANY data exists for it in this enrollment, even if no active instances exist, then yes.
+  #   (This supports both inactive instances and migrated-in data, as well as context changes such as change in HoH)
```

Comment:
I think the "inactive instances" part could be clarified. Maybe something like:


```
This allows users to see data in the following scenarios:
(1) data was previously collected in HMIS, but the rule (Instance) has since been deactivated
(2) data was previously collected for this Enrollment, but is no longer collected because of some change in context (e.g. they are no longer HoH, or the project funder/attributes changed)
(3) data was migrated-in, but has no rule (Instance) to enable it
```